<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main_window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="select_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Dir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="run_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Find String</string>
    </property>
   </widget>
   <widget class="QTableView" name="table_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>851</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="status_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>453</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Folder Prepared</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="interrupt_prep_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Interrupt Prep</string>
    </property>
   </widget>
   <widget class="QLabel" name="folder_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>14</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_bar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>661</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_edit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>581</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="interrupt_scan_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Interrupt Scan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prepare_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Prepare Dir</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
